tests/genai/judges/test_make_judge.py

```diff
@@ -3509,3 +3514,126 @@ def test_response_format_uses_generic_field_description(description):
     output_fields = judge.get_output_fields()
     result_field = next(f for f in output_fields if f.name == "result")
     assert result_field.description == _RESULT_FIELD_DESCRIPTION
+
+
+def test_make_judge_with_inference_params():
+    inference_params = {"temperature": 0.0, "max_tokens": 100}
+    judge = make_judge(
+        name="test_judge",
+        instructions="Check if {{ outputs }} is formal",
+        model="openai:/gpt-4",
+        inference_params=inference_params,
+    )
+
+    assert judge.inference_params == inference_params
+    assert judge._inference_params == inference_params
+
+
+def test_make_judge_without_inference_params():
+    judge = make_judge(
+        name="test_judge",
+        instructions="Check if {{ outputs }} is formal",
+        model="openai:/gpt-4",
+    )
+
+    assert judge.inference_params is None
+    assert judge._inference_params is None
+
+
+def test_instructions_judge_with_inference_params():
+    inference_params = {"temperature": 0.5, "top_p": 0.9}
+    judge = InstructionsJudge(
+        name="test_judge",
+        instructions="Check if {{ outputs }} is accurate",
+        model="openai:/gpt-4",
+        inference_params=inference_params,
+    )
+
+    assert judge.inference_params == inference_params
+
+
+def test_inference_params_passed_to_invoke_judge_model(mock_invoke_judge_model):
```

Comment:
can we merge this test with the first one? same with `test_inference_params_in_repr` with `test_make_judge_with_inference_params` and `test_inference_params_not_in_repr_when_none` with `test_make_judge_without_inference_params`. same with the serialization tests too